project/templates/header.html

```diff
@@ -17,25 +17,35 @@
 }
 </script>
 <div class="header">
-    <a href="/" class="logo">lusco-fusco</a>
-    <div style="float:right">
+    <!-- Logo -->
+    <a href="/" class="logo" ><image class="logo" src="/static/images/luscofusco_logotipo_B_Y.svg" alt="luscofusco logo"></image></a>
```

Comment:
Logo has a small yellow line that grows when you go over the image, like a link. I :heart: that animation. However, Has been that small yellow line make on purpose? or did it is inherit of link's class by error?

Reply:
Yeah! Line has been make for you ❤️ 
In fact, it was a mistake that i haven't see in my laptop. If you love that animation, i will try to do it well.

Reply:
Thanks :yum: